Editor/Animation/PathMappings.cs

```diff
@@ -274,6 +278,16 @@ private string MapPath(EditorCurveBinding binding)
             return newClip;
         }
 
+#if MA_VRCSDK3_AVATARS_3_5_2_OR_NEWER
+        private VRCAnimatorPlayAudio ApplyMappingsToPlayAudio(VRCAnimatorPlayAudio audio)
+        {
+            Debug.Log($"SourcePath: {audio.SourcePath}");
```

Comment:
Please remove the Debug.Log statements before merging (I know, I'm bad at it too sometimes... 😅 )